pkg/asset/installconfig/azure/mock/azureclient_generated.go

```diff
@@ -105,8 +106,23 @@ func (m *MockAPI) GetResourcesProvider(ctx context.Context, resourceProviderName
 	return ret0, ret1
 }
 
-// GetResourcesProvider indicates an expected call of GetResourcesProvider.
+// GetResourcesProvider indicates an expected call of GetResourcesProvider
 func (mr *MockAPIMockRecorder) GetResourcesProvider(ctx, resourceProviderNamespace interface{}) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "GetResourcesProvider", reflect.TypeOf((*MockAPI)(nil).GetResourcesProvider), ctx, resourceProviderNamespace)
 }
+
+// ValidateResourceSkuAvailability mocks base method
+func (m *MockAPI) ValidateResourceSkuAvailability(ctx context.Context, region, diskType string) (*compute.ResourceSku, error) {
```

Comment:
Are there any tests using these?

Reply:
Not yet since these functions are not being called. Will add dummy test cases for now and plan to clean up later